assets/js/no-in-person-locations-display.tsx

```diff
@@ -0,0 +1,9 @@
+interface NoInPersonLocationsDisplayProps {
+  address: string;
+}
+
+function NoInPersonLocationsDisplay({ address }: NoInPersonLocationsDisplayProps) {
+  return <h2 role="status">in_person_proofing.body.location.po_search.none_found {address}</h2>;
```

Comment:
I already had this on my branch. I had not pushed it up as I was waiting for library to avoid getting error about t not being defined.  Going to resolve.  Thank you